Apps/APN-UIKit/NotificationServiceExtension/NotificationService.swift

```diff
@@ -7,9 +7,9 @@ class NotificationService: UNNotificationServiceExtension {
     var bestAttemptContent: UNMutableNotificationContent?
 
     override func didReceive(_ request: UNNotificationRequest, withContentHandler contentHandler: @escaping (UNNotificationContent) -> Void) {
-        CustomerIO.initialize(siteId: BuildEnvironment.CustomerIO.siteId, apiKey: BuildEnvironment.CustomerIO.apiKey, region: Region.US) { config in
-            config.autoTrackPushEvents = true
-        }
+//        CustomerIO.initialize(siteId: BuildEnvironment.CustomerIO.siteId, apiKey: BuildEnvironment.CustomerIO.apiKey, region: Region.US) { config in
```

Comment:
This line is resulting in ` Ambiguous use of 'autoTrackPushEvents'` that i need to resolve

Reply:
I could be wrong, but I think this error is happening because the Swift compiler is seeing finding multiple `CustomerIO.initialize` functions that it could call. The compiler is not sure what one to choose, so it gives this error.

Reply:
@levibostian yup, before I dig deep can you think of any reason why this might be happening?